Apps/WebClient/src/ClientApp/src/models/credentialConnection.ts

```diff
@@ -0,0 +1,36 @@
+import { StringISODateTime } from "./dateWrapper";
+
+export interface CredentialConnection {
+    walletConnectionId: string;
+    hdid: string;
+    issuerConnectionid: string;
+    connectedDate: StringISODateTime;
+    disconnectedDate: StringISODateTime;
+    qrCode: string;
+    state: ConnectionState;
+    version: string;
+}
+
+enum ConnectionState {
+    Pending,
+    Connected,
+    Disconnected,
+}
+
+export interface WalletCredential {
+    credentialId: string;
+    walletConnectionId: string;
+    issuerCredentialId: string;
+    sourcetype: string;
```

Comment:
`sourcetype` should be `sourceType`.

Reply:
The names are based on the discussion we have on https://qslvic.visualstudio.com/Health%20Gateway/_workitems/edit/10605.